lib/reporters.js

```diff
@@ -37,6 +38,15 @@ function DataDogReporter(apiKey, appKey) {
             appKeyAuth: appKey
         }
     });
+    if (this.apiHost) {
+        // Strip leading `app.` from the site in case someone copy/pasted the
+        // URL from their web browser. More details on correct configuration:
+        // https://docs.datadoghq.com/getting_started/site/#access-the-datadog-site
+        this.apiHost = apiHost.replace(/^app\./i, '');
```

Comment:
The previous README used `app.datadoghq.com` in the README, which does not work. I’m not sure if it used to.

I tested this with the default site and with `us3.datadoghq.com`.